specification/summary-care-record.yaml

```diff
@@ -278,10 +278,10 @@ paths:
         ## Sandbox test scenarios
         You can test the following scenarios in our sandbox environment:
 
-        | Scenario                         | Request                                                                                                                             | Response                          |
-        | -------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------|---------------------------------- |
-        | Happy path                       | `composition.identifier`=`FA60BE64-1F34-11EB-A2A8-000C29A364EB` `composition.subject:Patient.identifier`=`9995000180`               | HTTP Status 200 - Full Bundle     |
-        | No results                       | `composition.identifier`=`81CC2DA0-8882-11EB-B538-0800200C9A66` `composition.subject:Patient.identifier`=`9693042298`               | HTTP Status 200 - Empty Bundle    |
+        | Scenario                         | Request                                                                                                         | Response                          |
+        | -------------------------------- | ----------------------------------------------------------------------------------------------------------------|---------------------------------- |
+        | Happy path                       | `composition.identifier`=`FA60BE64-1F34-11EB-A2A8-000C29A364EB` `Patient.identifier`=`9000000009`               | HTTP Status 200 - Full Bundle     |
```

Comment:
This accidentally overwrites changes from db5a9ec306776f78d2f5fedc2e65321c1f15efa5